stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package io.getstream.chat.ui.sample.feature.chat
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import io.getstream.chat.android.livedata.ChatDomain
+import io.getstream.chat.android.livedata.controller.ChannelController
+import io.getstream.chat.android.livedata.utils.Event
+
+class ChatViewModel(private val cid: String, private val chatDomain: ChatDomain = ChatDomain.instance()) : ViewModel() {
+
+    private val channelController: ChannelController = chatDomain.useCases.getChannelController(cid).execute().data()
```

Comment:
I remember that we agreed no to use execute in ViewModels

Reply:
`getChannelController` use case only creates `ChannelController` class, but agree let's not use execute